### DISCUSSION TITLE
Passar muito tempo em uma empresa é saudável para a carreira?

### DISCUSSION THREAD

**Discussion by azvini** [frontendbr/forum#2063] [Discussão]
Olá a todos,
hoje enquanto eu estava conversando com alguns recrutadores sobre algumas vagas de emprego que me interessaram todos fizeram o mesmo comentário sobre a minha trajetória profissional: "Vc está a 4 anos na mesma empresa? Isso pode ser mal visto pelos outros recrutadores". 
Isso me gerou uma dúvida muito grande, como o mercado enxerga esse ponto? Visto que eu estou na mesma empresa a 4 anos, e eu ainda vejo muito potencial de desenvolvimento pessoal/profissional lá dentro ficar muito tempo em um emprego é visto como algo ruim, ou foi só papo dos recrutadores para aumentar a possibilidade de eu aceitar a vaga de emprego deles?

**Comment by lddv** [comment]
> eu ainda vejo muito potencial de desenvolvimento pessoal/profissional lá dentro

Você poderia ter dado essa resposta ao recrutador. Acredito que o profissional é dono da sua própria carreira e se você se sente motivado para ficar na empresa, o recrutador deveria ficar feliz porque você potencialmente ficará muito tempo na empresa nova se decidir trocar de emprego.

Muito estranho esse relato, pois pra mim o sinal de alerta seria uma pessoa que pula de emprego em emprego e não para em lugar nenhum por mais de ~6 meses~ (coloque aqui o seu tempo mínimo para ficar em uma empresa e aprender sobre o negócio e tecnologias que ela utiliza).

Resumindo: confia na sua trajetória.

**Comment by wilsonneto-swe** [comment]
@azvini os recrutadores em questão querem que você saia para ir trabalhar com eles, a meta deles é contratar e te desmotivar de ficar na empresa atual.

Como o @lddv falou... Hoje estou envolvido no recrutamento de candidatos na avaliação técnica, e se vejo que nos últimos empregos o cara mal ficou um ano, eu descarto os currículos, se vejo que o cara está há mais de 3 anos na empresa atual ai eu tiro o chapéu e dou preferencia. Os teus primeiros 6 meses e até um ano, será um investimento da nova empresa, pois para você performar bem e de acordo com a cultura e regras de negócio da nova empresa, é só com um tempo de casa, no inicio você mal sabe todas as pontas que o código que você está mexendo afeta e nem mesmo as regras de negócios que levaram aquele código...

Se a equipe despendeu tempo para treinar e entrosar com um dev que ficará apenas 6 meses, considero contratação um fracasso, apenas atrasou a equipe... Então penso que quanto mais tempo de empresa você ter, melhor e mais comprometido você será visto por quem irá te contratar, aqueles que não tem metas de contratação, mas sim metas de entregas...

**Comment by MiqueiasGFernandes** [comment]
@azvini , Observação estranha mesmo, sempre vi com bons olhos quem está motivado pra permanecer bastante tempo com um mesmo time

**Comment by ninetails** [comment]
Olha, TI é uma área de alta rotatividade, rápido crescimento pessoal e profissional e de aprendizagem. E nisso, polarizando o assunto (claro que deve existir um espectro no meio, mas a gente adora dualizar), existem dois perfis de "recrutadores" relacionados ao assunto (não só RH mas por exemplo, equipe técnica que procuram por profissionais):

* Aquele que acha que se a pessoa não mudou de emprego em 1 ano, não aproveita novas oportunidades de crescimento;
* Se a pessoa ficou 5 anos numa mesma empresa, ela não sairá da minha por qualquer coisinha.

O que eu quero ressaltar nisso é que os dois não são opostos. Ficando muito tempo na empresa vc perde a oportunidade de ter _upgrades_ salariais e de carreira de forma rápida. Ficar "pulando de galho em galho", pro recrutador parece que você sairá na primeira oportunidade de melhoria nestes quesitos, só que ao mesmo tempo eles precisam de pessoas que fiquem na empresa pelo máximo do tempo pois muito se perde em ter que treinar novos profissionais e fazer todo o _onboarding_ e disseminar conhecimento. Também tem a questão de _budget_ limitado. Porém isso também abre outra discussão relacionada a mérito, sobre não pagar (ou poder pagar) o quanto o profissional vale e já foi discutido em outras _threads_ no fórum.

Qual o ponto em comum dos dois, já que não são opostos? Se você não _subiu_ na sua carreira, esteve estagnado numa posição dentro duma empresa quer dizer que você não cresceu como profissional. E no LinkedIn ou currículo, nós desenvolvedores dificilmente ganhamos "prêmios" que justifiquem que o que fazemos de excelência ou que cada dia fazemos melhor, não temos premiações em Cannes etc. Logo no currículo ficará só um "ficou X anos na empresa" não mostrando sua evolução ou excelência dentro dela. E se depois de 4 anos você ainda vê potencial, para alguns recrutadores pode também soar que você não sugou tudo o que a empresa tinha a lhe ensinar nesses 4 anos.

Mudar de emprego por mudar também não é a questão, é o errado. O que vc precisa saber comunicar tanto no LinkedIn quanto quando lhe perguntarem é o quanto de conhecimento você acumulou nesse tempo, seja numa só empresa, seja trabalhando em diversas e o que  essa expertise pode agregar na empresa contratante. Se você não soube o que responder a estes recrutadores ou está na dúvida, tá na hora de ou procurar novo emprego ou aumentar tuas responsabilidades na empresa atual, crescer de cargo e correr pra absorver tudo o que ainda vc vê de potencial de crescimento.

Ficar 4 anos na empresa não é uma crítica se você tiver na ponta da língua como contra-argumentar sem soar negativamente pra sua carreira.
